microservices/time-series-analytics/src/opcua_alerts.py

```diff
@@ -89,9 +89,9 @@ async def connect_opcua_client(self, secure_mode, max_retries=10):
             try:
                 if secure_mode.lower() == "true":
                     kapacitor_cert = ("/run/secrets/"
-                                    "time_series_analytics_microservice_Server_server_certificate.pem")
+                                    "server_certificate.pem")
```

Comment:
As mentioned in the other PR, let's use client_certificate and client_key instead.
I'm just thinking instead of hardcoding, how about we have this as ENV vars and use it in the docker compose volume mounts the same ENVs, this would make the things more flexible.